wurst.build

```diff
@@ -10,12 +10,13 @@ buildMapData:
   scenarioData:
     description: Island Troll Tribes is a PvP-centric survival map in which multiple
       tribes of trolls compete to gather resources, craft equipment, and eradicate
-      each other. Join us at https://discord.gg/hYzTRYf.
+      each other. Join our Discord server on https://discord.gg/Rh9JdKs for mustering lobbies, reporting bugs, and making
```

Comment:
I think it should be `at https...`. Also, did Wurst not automatically resize this when running? I left mine as a really long line and it formatted the description into 3 lines instead.

Reply:
I noticed this too, but it looks like it doesn't resize the lines on my side.